plaso/cli/image_export_tool.py

```diff
@@ -91,6 +93,8 @@ def __init__(self, input_reader=None, output_writer=None):
     self._filter_file = None
     self._path_spec_extractor = extractors.PathSpecExtractor()
     self._process_memory_limit = None
+    self._report_output = collections.defaultdict(list)
+    self._report_writer = None
```

Comment:
I don't see this being used - remove?

Reply:
Yeah, that's and an old leftover. Thanks for spotting.